src/rgw/rgw_op.cc

```diff
@@ -2547,7 +2550,11 @@ int RGWCreateBucket::verify_permission()
     return -EACCES;
   }
 
-  if (!verify_user_permission(this, s, ARN(s->bucket), rgw::IAM::s3CreateBucket)) {
+  rgw_bucket bucket;
+  bucket.name = s->bucket_name;
+  bucket.tenant = s->bucket_tenant;
+  ARN arn = ARN(bucket);
```

Comment:
Rather than doing this, it might be better to just add a static ARN::make_bucket_arn that takes two string-views.

Reply:
ok